.scripts/cherrypick.sh

```diff
@@ -0,0 +1,12 @@
+# WARNING: DO NOT EDIT!
+#
+# This file was generated by plugin_template, and is managed by it. Please use
+# './plugin-template --github pulp_npm' to update this file.
+#
+# For more info visit https://github.com/pulp/plugin_template
+
+pip install PyGithub GitPython yq
```

Comment:
I'd argue that perhaps we should dump these into the ```dev_requirements.txt``` file and just assume they're installed.  No strong feelings though.